src/checks/prohibeted-networks.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+# This feature will check if the NS server IP is global or private
+# If it is global it will pass the test
+# If it is local it will prompt the user that this IP is from a private network and
+# test will fail
+
+# Variables:
+# ns_server --> should be taken as an argument from the command line
+# ip_val --> extracted value of the ns server
+import dns.resolver
+from IPy import IP
+import ipaddress
+
+# Hard coded special IPs that the imports don't cover
+bad_1 = "203.0.113.0"
+bad_2 = "224.0.0.0"
+
```

Comment:
It checks for 203.0.113.0 only. On the IANA web page the prohibited network is 203.0.113.0/24. That is, the range 203.0.113.0 - 203.0.113.255.
Same for bad_2

Reply:
OK. I will search for a way to solve that.